MIAUI/Model/User.cs

```diff
@@ -1,6 +1,16 @@
-﻿namespace MIAUI.Model;
+﻿using SQLite;
+
+namespace MIAUI.Model;
+
+[Table("user")]
 public class User
 {
+    [PrimaryKey, AutoIncrement, Column("_id") ]
+    public int Id { get; set; }
+
+    [MaxLength(10), Unique]
```

Comment:
10 seems too short for a name - For example "Giovanny Hernandez" is already 18 characters